tests/settings_tests/tests.py

```diff
@@ -445,3 +447,25 @@ def test_tuple_settings(self):
             finally:
                 del sys.modules['fake_settings_module']
                 delattr(settings_module, setting)
+
+
+class TestDatabaseMirror(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_without_mirror(self):
+        Simple.objects.using('default').create()
+        Simple.objects.using('other').create()
+
+        self.assertEqual(Simple.objects.count(), 2)
+
+    def test_with_mirror(self):
+        databases = settings.DATABASES
+        databases['other']['TEST']['MIRROR'] = 'default'
+
+        with override_settings(DATABAES=databases):
```

Comment:
You have a spelling error here. It's `DATABASES`. 

Even if you correct that, this would raise a warning:

```
tests/settings_tests/tests.py:472: UserWarning: Overriding setting DATABASES can lead to unexpected behavior.
  with override_settings(DATABASES=databases):
```

I'm not at sure you can use this here. At the very least it would need to decorate the test method or whole test case, but the databases are set up at the beginning of the test run, so instead of using `override_settings` I think you will need to declare a new alias just to serve as the mirror here.